src/tools/sfdk/command.cpp

```diff
@@ -111,15 +111,26 @@ class PropertiesAccessor
     {
         return tr("Value too big");
     }
+
     static QString valueCannotBeDecreasedMessage()
     {
         return tr("Value cannot be decreased");
     }
 
+    static QString valueCannotBeIncreasedMessage()
+    {
+        return tr("Value cannot be increased");
+    }
+
     static QString unknownPropertyMessage(const QString &name)
     {
         return tr("Unrecognized property \"%1\"").arg(name);
     }
+
+    static QString modifyNotSupportedMessage(const QString &name)
+    {
+        return tr("Virtual machine does not support modifying property \"%1\"").arg(name);
```

Comment:
I would prefer using more general (reusable) message, e.g.

```
static QString readOnlyPropertyMessage()
{
    return tr("Read-only property");
}
```

Could you please also fix the error reporting by adding property name prefix where it is written to stderr in `BuiltinWorker::setProperties`?

```
qerr() << property << ": " << errorString << endl;
```
...and removing the `name` argument from `unknownPropertyMessage`.

(at least I think that would work well after brief check)

Reply:
fixed. Seems to work.